doc/modules/model_evaluation.rst

```diff
@@ -1401,7 +1401,11 @@ to the given limit.
     .. [FC2009] Ferri, Cèsar & Hernandez-Orallo, Jose & Modroiu, R. (2009). 
        `An Experimental Comparison of Performance Measures for Classification. 
        <https://www.math.ucdavis.edu/~saito/data/roc/ferri-class-perf-metrics.pdf>`_
-       Pattern Recognition Letters. 30. 27-38. 
+       Pattern Recognition Letters. 30. 27-38.
+
+    .. [PD2000] Provost, F., Domingos, P. (2000). Well-trained PETs: Improving
```

Comment:
Its interesting how the Provost 2000 reference mentions the weighting in section 6.2 http://pages.stern.nyu.edu/%7Efprovost/Papers/pet-wp.pdf 

> For datasets with more than two classes we computed the expected AUC, which is the weighted average average of the AUCs.... The weight of a class's AUC is the class's frequency in the data.

Given this took a little bit of searching to find, may we include the section as well in the reference?

Reply:
I've done that now. I wasn't quite sure whether there is an expected citation style for that, so I just took a stab at it.